azure-local/update/update-best-practices.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Best Practices for managing Azure Local Update Management
+description: Learn the best practices for managing Azure Local environments.
+author: alkohli
+ms.topic: overview
+ms.date: 09/03/2025
+ms.author: alkohli
+ms.reviewer: alkohli
+ms.service: azure-local
+---
+
+# Best practices for managing Azure Local environments
+
+This article provides an overview of Azure Local update management, including best practices and common pitfalls to help keep Azure Local secure, up to date, and compliant. It is intended for IT decision-makers, infrastructure architects, and operations teams responsible for Azure Local deployments.
+
+## Understand Azure Local update management
+
+Staying current with security fixes and feature improvements is important for all Azure Local components. For an overview of Azure Local updates, see [About updates for Azure Local](./about-updates-23h2.md).
+
+Azure Local supports update management through Azure Update Manager (AUM), which provides centralized lifecycle management. With AUM, you can apply, view, and manage updates across all Azure Local instances from a single interface.
+
+:::image type="content" source="./media/update-best-practices/updates.png" alt-text="Diagram that describes the key features of Azure Local update management." lightbox="./media/update-best-practices/updates.png":::
+
+## Do's for Azure Local updates
+
+Follow these practices to ensure smooth, reliable updates for Azure Local instances deployed at scale.
+
+1. **Use Azure Update Manager.**
+
+   AUM provides a centralized view to apply and manage updates across all Azure Local instances. In the Azure portal, go to **Azure Update Manager** > **Resources** > **Azure Local**.
+
+   :::image type="content" source="./media/update-best-practices/azure-update-manager.png" alt-text="Screenshot of the Azure Update Manager displaying the Azure Local systems ready for updates. ." lightbox="./media/update-best-practices/azure-update-manager.png":::
+
+   - **Use filters to identify Azure Local resources ready for updates.**
+
+      Use filters like:
+      - Update Readiness = Healthy
+      - Status = Updates available
+
+   - **Batch updates for large environments.**
+
+      Group clusters using:
+      - Tags
+      - Resource group
+      - Subscription
+      - Current version
+      - Location
+   
+      Define a model where you are updating in chunks.  
+
+   - **Test environment strategy.**
```

Comment:
put "test environment strategy" as 1st bullet point

Reply:
Moved Test environment strategy as the first bullet under AUM.